### PR TITLE
B2B-1544 Added note about gift cards, etc with PO Approval

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) notes that gift carts, reward points and store credit is not supported for online payments with POs.

## Magento release version

- 2.4.2

## Product editions

- B2B extension

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-dashboard-approval-rules.html

### CODE REVIEW

File: src/customers/account-dashboard-approval-rules.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ _Company Approval Rules_
...
 Depending on the company role and permissions, users may have the option to edit, delete, or view approval rules.
 
+## Purchase order payment methods
+
+Purchase order approval supports both online and offline payment methods.
+
+All default offline payment methods are supported for purchase order approvals.
+For online payments, the following methods are supported:
+
+* Paypal Express
```
Comment (dhaecker) [inline src/customers/account-dashboard-approval-rules.md:29]: @dobooth we should indicate somewhere that we only support `In-Context Checkout Experience`. out-of-context works mostly but there are still issues & we dont support it in POs yet

File: src/customers/account-dashboard-approval-rules.md [hunk 2]
```diff
@@ -19,6 +19,18 @@ _Company Approval Rules_
...
+* Braintree payments
+
+Note that using gift cards, store credit, or reward points with online payment methods for purchase orders is not supported. Enabling these features with online payments may cause some unexpected behavior. It is recommended to disable gift cards, store credit, and reward points when online payments are enabled for purchase orders.
+
```
Comment (dhaecker) [inline src/customers/account-dashboard-approval-rules.md:33]: should also indicate that uusers can bypass POs w/ paypal if the merchant has paypal enabled to show in the minicart, product details page, or somewhere other than the checkout payment page

also include that changes to discounts, catalog price rules, shipping, etc will not propagate to a PO after creation but before payment

**Review by dhaecker — CHANGES_REQUESTED** [review]

File: src/customers/account-dashboard-approval-rules.md (hunk 1 above)
Comment (dobooth) [reply src/customers/account-dashboard-approval-rules.md:29]: Done

File: src/customers/account-dashboard-approval-rules.md (hunk 2 above)
Comment (dobooth) [reply src/customers/account-dashboard-approval-rules.md:35]: We have a KB for that.

**Comment by dobooth** [comment]
Closing because I made it from the wrong branch.

**Pull request closed without merging**